src/helpers.js

```diff
@@ -101,6 +101,17 @@ const getEndpoints = ({ collectionTypes, singleTypes }, schemas) => {
       const options = types.find((config) => config.singularName === singularName);
       const { queryParams, queryLimit } = options;
 
+      if (uid === 'plugin::upload.file') {
+        return {
+          singularName,
+          pluralName,
+          kind,
+          uid,
+          endpoint: '/api/upload/files',
+          queryParams: {},
```

Comment:
Is it necessary to block user query params here? I don't know if it's possible today, but at some point we may have images with support for localization or draft & publish

Reply:
The API response is not the same for the files and the file model does not create any children node so for the moment I have jsut added the support and I'll let the expansion deal with what you think is best.